### PR TITLE
Organize source into subpackages

### PR DESCRIPTION
**Pull request by kgoins**

### CODE REVIEW

**Comment by kgoins** [comment]
This should make it easier to selectively import functionality from this project into another :)

**Comment by kgoins** [comment]
Absolutely open to feedback on structure etc

**Comment by zellyn** [comment]
@srlehn any feelings on this PR? It had accidentally slipped though without me noticing (sorry, @kgoins!).

**Comment by kgoins** [comment]
@srlehn excellent feedback. Good catch on the misnamed import path, noob mistake. I'll re-sync with master then RFC after changes have been made.

**Comment by srlehn** [comment]
**findCookie**() shouldn't be made public and instead be removed as it is currently unused.
I think Filters provide more flexibility.

**Comment by srlehn** [comment]
Hi @kgoins I updated your PR (https://github.com/kgoins/kooky/pull/4) with the changes that happened since and did a few changes for the filters I introduced and some other small adjustments.

@zellyn I do have some browsers waiting to be added. The separation would be a good thing if more additions happen. The code I have for the Edge browser adds another dependency again for ESE.

File: pkg/chrome/chrome_linux.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ import (
 
 	"golang.org/x/crypto/pbkdf2"
 
-	"github.com/zalando/go-keyring/secret_service"
+	ss "github.com/zalando/go-keyring/secret_service"
```
Comment (zellyn) [inline pkg/chrome/chrome_linux.go:23]: Is this necessary?

**Review by zellyn — CHANGES_REQUESTED** [review]
Agree with all @srlehn's comments…

File: pkg/chrome/chrome_linux.go (hunk 1 above)
Comment (srlehn) [reply pkg/chrome/chrome_linux.go:23]: The package is actually called that way grafik. Showing this with the alias makes it clear.

Im not a fan of that abbreviation though (nazi related)  - I'll rename it to the url-end (secret_service).

**Comment by zellyn** [comment]
I think you need to send a PR to me… I only see the original changes here

**Comment by srlehn** [comment]
https://github.com/zellyn/kooky/pull/22

**Pull request merged**